client/signup/p2-processing-screen/index.jsx

```diff
@@ -23,15 +23,17 @@ function P2SignupProcessingScreen() {
 			</div>
 
 			<div className="p2-processing-screen__text">
-				{ translate( '{{h2}}Hooray!{{br/}}Your new P2 is{{br/}}almost ready.{{/h2}}', {
+				{ translate( '{{h2}}Hooray!{{/h2}} {{p}}Your new P2 is almost ready.{{/p}}', {
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.